src/main.cpp

```diff
@@ -145,6 +156,12 @@ void loop()
     yield();
     WebApi.loop();
     yield();
+    if (display_type != display_type_before){

```

Comment:
Erstelle eine OledDisplay.loop() methode und übernimm alles dorthin. (vgl. alle anderen Aufrufe in der loop function)